testing/tests/DevExpress.ui.widgets.dataGrid/dataController.tests.js

```diff
@@ -632,27 +632,28 @@ QUnit.test("Using focusedRowEnabled should set sorting for the not sorted simple
     // act
     this.dataController._refreshDataSource();
     // assert
-    assert.equal(dataSource.items()[0].name, "Alex", "Item name is Alex");
-    assert.equal(dataSource.items()[1].name, "Alice", "Item name is Alice");
+    assert.equal(dataSource.items()[0].name, "Alex", "Item0");
+    assert.equal(dataSource.items()[1].name, "Dan", "Item1");
     // act
     dataSource.pageIndex(1);
     dataSource.load();
-    assert.equal(dataSource.items()[0].name, "Bob", "Item name is Bob");
-    assert.equal(dataSource.items()[1].name, "Dan", "Item name is Dan");
+    assert.equal(dataSource.items()[0].name, "Bob", "Item2");
+    assert.equal(dataSource.items()[1].name, "Alice", "Item3");
 });
 
-QUnit.test("Using focusedRowEnabled should set sorting for the not sorted composite key columns", function(assert) {
+QUnit.test("Using focusedRowEnabled should not set sorting for the not sorted composite key columns", function(assert) {
```

Comment:
Enable remoteOperations in this test to check composite key scenario with sorting by key